Library/Formula/texinfo.rb

```diff
@@ -28,8 +28,8 @@ def install
     # However, it complains about installing epsf.tex in TEXMF/tex/generic/dvips, so let's do that...
     # This somewhat breaks the homebrew philosophy, I am sorry.
     # Also, we don't depend on tex-live, but this directory only exists if it is installed.
-    if File.exist? "#{HOMEBREW_PREFIX}/share/texmf-dist/" then
-      cp "doc/epsf.tex", "#{HOMEBREW_PREFIX}/share/texmf-dist/tex/generic/dvips/"
+    if File.exist?(HOMEBREW_PREFIX/"share/texmf-dist/")
+      cp "doc/epsf.tex", HOMEBREW_PREFIX/"share/texmf-dist/tex/generic/dvips/"
```

Comment:
Pretty sure the answer is yes here; I just wasn't sure how to fix it best. I guess we install the `epsf.tex` file to `libexec` and then move it `post_install` or something?